trunk/SUAVE/Methods/Weights/Buildups/Common/prop.py

```diff
@@ -0,0 +1,270 @@
+    seg.append(box[box[:,1]>np.mean(box[:,1])]*chord)   # Upper Fwd Segment
+    seg.append(box[box[:,1]<np.mean(box[:,1])]*chord)   # Lower Fwd Segment
+
+    # Flap & Drag Inertia
+
+    capInertia = 0
+    capLength = 0
+
+    for i in range(0,2):
+        l = np.sqrt(np.sum(np.diff(seg[i],axis=0)**2,axis=1))   # Segment Lengths
+        c = (seg[i][1::]+seg[i][0::-1])/2                       # Segment Centroids
+
+        capInertia += np.abs(np.sum(l*c[:,1] **2))
+        capLength  += np.sum(l)
+
+    # Shear Properties
+
+    box = coord
+    box = box[box[:,0]<=fwdWeb[1]]
+    z = box[box[:,0]==fwdWeb[0],1]*chord
+    shearHeight = np.abs(z[0] - z[1])
+
+    # Core Properties
+
+    box = coord
+    box = box[box[:,0]>=fwdWeb[0]]
+    box = box*chord
+    coreArea = 0.5*np.abs(np.dot(box[:,0],np.roll(box[:,1],1))-
+        np.dot(box[:,1],np.roll(box[:,0],1)))   # Shoelace Formula
+
+    # Shear/Moment Calculations
+
+    Vz = np.append(np.cumsum(( F[0:-1]*np.diff(x))[::-1])[::-1],0)  # Bending Moment
+    Mx = np.append(np.cumsum((Vz[0:-1]*np.diff(x))[::-1])[::-1],0)  # Torsion Moment
+    My = np.append(np.cumsum(( Q[0:-1]*np.diff(x))[::-1])[::-1],0)  # Drag Moment
+
+#-------------------------------------------------------------------------------
+# Mass Calculation
+#-------------------------------------------------------------------------------
+
+    while error > tolerance:
+        CF = (SF*omega**2*
+            np.append(np.cumsum(( m[0:-1]*np.diff(x)*x[0:-1])[::-1])[::-1],0))  # Centripetal Force
+
+        # Calculate Skin Weight Based on Torsion
+
+        tTorsion = My/(2*bidirectional_carbon_fiber().ultimate_shear_strength*enclosedArea)                          # Torsion Skin Thickness
+        tTorsion = np.maximum(tTorsion,bidirectional_carbon_fiber().minimum_gage_thickness*np.ones(N))           # Gage Constraint
+        mTorsion = tTorsion * skinLength * bidirectional_carbon_fiber().density                  # Torsion Mass
+
+        # Calculate Flap Mass Based on Bending
+
+        tFlap = CF/(capLength*unidirectional_carbon_fiber().ultimate_tensile_strength) +   \
+            Mx*np.amax(np.abs(box[:,1]))/(capInertia*unidirectional_carbon_fiber().ultimate_tensile_strength)
+        mFlap = tFlap*capLength*unidirectional_carbon_fiber().density
+        mGlue = epoxy().minimum_gage_thickness*epoxy().density*capLength*np.ones(N)
+
+        # Calculate Web Mass Based on Shear
+
+        tShear = 1.5*Vz/(bidirectional_carbon_fiber().ultimate_shear_strength*shearHeight)
+        tShear = np.maximum(tShear,bidirectional_carbon_fiber().minimum_gage_thickness*np.ones(N))
+        mShear = tShear*shearHeight*bidirectional_carbon_fiber().density
+
+        # Paint Weight
+
+        mPaint = skinLength*paint().minimum_gage_thickness*paint().density*np.ones(N)
+
+        # Core Mass
+
+        mCore = coreArea*honeycomb().density*np.ones(N)
+        mGlue += epoxy().minimum_gage_thickness*epoxy().density*skinLength*np.ones(N)
```

Comment:
unpack things

Reply:
Changed to unpack of material properties.